cmd/juju/cloud/add_test.go

```diff
@@ -201,7 +212,7 @@ func (s *addSuite) TestAddExisting(c *gc.C) {
 	fake.Call("PersonalCloudMetadata").Returns(mockCloud, nil)
 
 	_, err = s.runCommand(c, fake, "homestack", cloudFile.Name())
-	c.Assert(err, gc.ErrorMatches, "\"homestack\" already exists; use `update-cloud` to replace this existing cloud")
+	c.Assert(err, gc.ErrorMatches, "use `update-cloud homestack --local` to override known definition: \"homestack\" already exists")
```

Comment:
The user didn't type "--local" as a cmd arg, and yet we are prompting then to do so, but in reality --local is not needed to fix the error; what is needed is the use of "update" rather than "add". Including --local in the error message incorrectly implies to the user that is necessary for the fix